.openshift-ci/gcp/delete-vm.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e
+
+deleteGCPVM() {
+    local GCP_VM_NAME="$1"
+    shift
+
+    [ -z "$GCP_VM_NAME" ] && \
+        echo "error: missing parameter GCP_VM_NAME" && return 1
+
+    success=false
+    for _ in {1..3}; do
+        if gcloud compute instances delete "$GCP_VM_NAME";
+        then
+            success=true
+            break
+        fi
+    done
+
+    if test ! "$success" = "true"; then
```

Comment:
I see this pattern has made its way past to some other scripts, could we double check we change these too?
```suggestion
    if [[  "$success" != "true" ]]; then
```